packages/arb-token-bridge-ui/src/components/TransferPanel/hooks/useAmountBigNumber.ts

```diff
@@ -10,15 +11,22 @@ import { useSelectedTokenDecimals } from '../../../hooks/TransferPanel/useSelect
 
 export function useAmountBigNumber() {
   const [{ amount }, setQueryParams] = useArbQueryParams()
+  const debouncedAmount = useDebounce(amount, 300)
```

Comment:
I would not debounce this hook. If it needs to be debounce somewhere, then the caller should debounce it